<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormStandardCategoryDetails</class>
 <widget class="QDialog" name="FormStandardCategoryDetails">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>209</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>180</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="m_lblParentCategory">
       <property name="text">
        <string>Parent category</string>
       </property>
       <property name="buddy">
        <cstring>m_cmbParentCategory</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="m_cmbParentCategory">
       <property name="toolTip">
        <string>Select parent item for your category.</string>
       </property>
       <property name="iconSize">
        <size>
         <width>13</width>
         <height>12</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="m_lblTitle">
       <property name="text">
        <string>Title</string>
       </property>
       <property name="buddy">
        <cstring>m_txtTitle</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="m_lblDescription">
       <property name="text">
        <string>Description</string>
       </property>
       <property name="buddy">
        <cstring>m_txtDescription</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="m_lblIcon">
       <property name="text">
        <string>Icon</string>
       </property>
       <property name="buddy">
        <cstring>m_btnIcon</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QToolButton" name="m_btnIcon">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Select icon for your category.</string>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="popupMode">
        <enum>QToolButton::InstantPopup</enum>
       </property>
       <property name="autoRaise">
        <bool>false</bool>
       </property>
       <property name="arrowType">
        <enum>Qt::NoArrow</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="LineEditWithStatus" name="m_txtTitle" native="true"/>
     </item>
     <item row="2" column="1">
      <widget class="LineEditWithStatus" name="m_txtDescription" native="true"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="m_buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineEditWithStatus</class>
   <extends>QWidget</extends>
   <header>lineeditwithstatus.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>m_buttonBox</tabstop>
  <tabstop>m_cmbParentCategory</tabstop>
  <tabstop>m_btnIcon</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>m_buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FormStandardCategoryDetails</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>170</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
